packages/transformer/README.md

```diff
@@ -1,3 +1,5 @@
+⚠️ Although the latest version of this package declares a dependency on PixiJS 7, it now has a bug where pointer events outside of the handles causes the UI to stop responding. This is because of how PixiJS (EventSystem) now excludes events outside of a DisplayObject. See (#73)
```

Comment:
```suggestion
⚠️ Although the latest version of this package declares a dependency on PixiJS 7, it now has a bug where pointer events outside of the handles causes the UI to stop responding. This is because of how PixiJS (EventSystem) now excludes events outside of a DisplayObject. See (#71)
```